pkg/cache/placeholder.go

```diff
@@ -104,3 +108,20 @@ func (p *Placeholder) String() string {
 	return fmt.Sprintf("appID: %s, taskGroup: %s, podName: %s/%s",
 		p.appID, p.taskGroupName, p.pod.Namespace, p.pod.Name)
 }
+
+func getPlaceHolderImage() string {
+	if placeHolderImage == "" {
+		imageEnvSetting := os.Getenv(constants.PlaceHolderImageEnvVar)
+		if imageEnvSetting != "" {
+			placeHolderImage = imageEnvSetting
+			log.Logger().Info("Using placeholder image from environment variable", zap.String("imageName",
```

Comment:
I think we don't need to log here but can just use the config printout that comes in from the shim startup already.